hw/hw12/app/migrations/0001_initial.py

```diff
@@ -0,0 +1,119 @@
+            name="Student",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("first_name", models.CharField(max_length=30)),
+                ("last_name", models.CharField(max_length=30)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Teacher",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("first_name", models.CharField(max_length=30)),
+                ("last_name", models.CharField(max_length=30)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="HomeworkResult",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("solution", models.TextField()),
+                ("end_time", models.DateTimeField()),
+                (
+                    "homeworker",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="app.student"
+                    ),
+                ),
+                (
+                    "task_text",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="app.homework"
+                    ),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="HomeworkDone",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "good_homework",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="app.homeworkresult",
+                    ),
+                ),
+            ],
+        ),
+        migrations.AddField(
+            model_name="homework",
+            name="teacher",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, to="app.teacher"
+            ),
+        ),
+    ]
```

Comment:
There are too many blocks of nested code, can we use configs here to reduce lines count and complexity?